### PR TITLE
Embedded the Build Status for the Repository in Readme

### PR DESCRIPTION
**Pull request by sawood14012**
I am adding the QA checks and then will add this repository to the QA dashboard Meanwhile have just embedded the Build Status in Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Dependency Analytics LSP Server
 
+[![Build Status](https://ci.centos.org/job/devtools-fabric8-analytics-lsp-server/100/badge/icon)](https://ci.centos.org/job/devtools-fabric8-analytics-lsp-server/100/)
```
Comment (arajkumar) [inline README.md:3]: Link is incorrect, it is pointing a specific build(100).

Look at https://ci.centos.org/job/devtools-fabric8-analytics-lsp-server/badge/ for the right link.

[Build Status](https://ci.centos.org/job/devtools-fabric8-analytics-lsp-server/)

Comment (sawood14012) [reply README.md:3]: ohh sorry will change it

Comment (sawood14012) [reply README.md]: @arajkumar PTAL

**Comment by arajkumar** [comment]
[test]

**Comment by sawood14012** [comment]
@arajkumar i have updated it to the master build satatus PTAL

**Pull request merged**